### PR TITLE
[DOCS] Add ESP32S3 to libraries support list

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
## Description of Change
Added ESP32S3 to libraries support list.
Added SDMMC to the list.

### CODE REVIEW

**Comment by P-R-O-C-H-Y** [comment]
@pedrominatel As you can see I have added the S3 to the table, but when is "Not Supported" in cell it it doesn't look good. Can we do something with that? :) Thanks for tips.

**Review by pedrominatel — APPROVED** [review]
LGTM

File: docs/source/libraries.rst [hunk 1]
```diff
@@ -9,53 +9,53 @@ Supported Peripherals
...
-| USB           | Not Supported | Yes           | Yes           | ESP32-C3 only CDC/JTAG        |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Wi-Fi         | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | ESP32-S3      | Comments               |
++===============+===============+===============+===============+===============+========================+
+| ADC           | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Bluetooth     | Yes           | Not Supported | Not Supported | Not Supported | Bluetooth Classic      |
```
Comment (SuGlider) [inline docs/source/libraries.rst:17]: ESP32-S3 supports BLE, the same for ESP32-C3
We can replace Yes and Not Supported by "BLE and Classic" or "BLE only".
Remove the comments - "Classic"

File: docs/source/libraries.rst [hunk 2]
```diff
@@ -9,53 +9,53 @@ Supported Peripherals
...
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Ethernet      | Yes           | Not Supported | Not Supported | Not Supported | (*)                    |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| GPIO          | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Hall Sensor   | Yes           | Not Supported | Not Supported | Not Supported |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| I2C           | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| I2S           | Yes           | No            | No            | Yes           | WIP                    |
```
Comment (SuGlider) [inline docs/source/libraries.rst:31]: I think that ESP32-S2 has I2S support... please check with @PilnyTomas

**Review by SuGlider — COMMENTED** [review]
LGTM! Please just check the I2S note that I left.

File: docs/source/libraries.rst (hunk 2 above)
Comment (P-R-O-C-H-Y) [reply docs/source/libraries.rst]: Fixed :) All chips has I2S supported.

Comment (SuGlider) [reply docs/source/libraries.rst]: I saw that the I2S example excludes the C3 with the .skip.esp32c3 tag file. Maybe Pilny's library doesn't support this SoC.

Comment (P-R-O-C-H-Y) [reply docs/source/libraries.rst]: I got info from him that it supports :) but C3 has only 1 I2S. Maybe the example is made using both I2S peripherals :)

**Pull request merged**